src/ds_caselaw_utils/data/court_names.yaml

```diff
@@ -269,13 +269,6 @@
       end_year: 2022
       selectable: false
       listable: true
-    - code: EWHC-Chancery-Patents
-      grouped_name: Patents Court
-      name: High Court (Patents Court)
-      link: https://www.gov.uk/courts-tribunals/patents-court
-      ncn: \[(\d{4})\] (EWHC) (\d+) \((Pat)\)
-      selectable: false
```

Comment:
Does the order in which they were listed in the YAML matter?

That is, given that the version with listable and selectable `false` was lower in the yaml, is there a chance that the Patents court will suddenly be selectable and listable where it wasn't before?

Not that it would be a bad thing, just checking if anyone needs to be informed so it's not a surprise to them. (It could be that the order didn't matter, or that the selectable and listable item was loaded first, in which case no change.)

Reply:
Patents was visible in the website, so I suspect this shouldn't change behaviour.